packages/tupaia-web/src/features/Map/MapOverlays/DataVisualsLayer.tsx

```diff
@@ -26,6 +26,7 @@ import {
 import { useMapOverlayReport } from '../utils';
 import { EntityCode } from '../../../types';
 import { processMeasureData } from './processMeasureData';
+import { MapTableModal } from '../MapTableModal/MapTableModal';
```

Comment:
Ideally imports like this would be like `import { MapTableModal } from './MapTableModal'` because we would export anything from an index file. However I think we don't need `MapTableModal` to be a folder in this case, because there is only one file, so it can just sit as a single file within `Map` folder, or, even better, inside `MapOverlaySelector` since it relates to that.